### ISSUE TITLE
Terraform version as an environment variable

### ISSUE THREAD

**Issue by jackivanov** [tfutils/tfenv#165]
There should be a way to specify terraform version as an environment variable, eg:

```
$ TFENV_TERRAFORM_VERSION=0.11.13 terraform version
Terraform v0.11.13

Your version of Terraform is out of date! The latest version
is 0.12.20. You can update by downloading from www.terraform.io/downloads.html
```

```
$ TFENV_TERRAFORM_VERSION=0.12.20 terraform version
Terraform v0.12.20
```

**Comment by Zordrak** [comment]
Seems reasonable. Shall consider it once 2.0.0 is out the door

**Comment by Zordrak** [comment]
Resolved by #222
